src/PowerShell/Microsoft.WinGet.Client.Engine/Properties/Resources.resx

```diff
@@ -189,9 +189,9 @@
   <data name="RepairAppExecutionAliasMessage" xml:space="preserve">
     <value>The App Execution Alias for the Windows Package Manager is disabled. You should enable the App Execution Alias for the Windows Package Manager. Go to App execution aliases option in Apps &amp; features Settings to enable it.</value>
   </data>
-  <data name="MicrosoftUIXaml27Message" xml:space="preserve">

```

Comment:
Should we switch this to be version agnostic?  Such as a format string and use the non-localizable package name value in it, like:
```
Dependency is not installed: {0}
```
Where `{0}` becomes the appropriate package name value that would be useful in further inspection (such as passing to `Get-AppxPackage` to confirm).